sklearn/metrics/_classification.py

```diff
@@ -3239,6 +3395,16 @@ def brier_score_loss(
         * otherwise, `pos_label` defaults to the greater label,
           i.e. `np.unique(y_true)[-1]`.
 
+    labels : array-like of shape (n_classes,), default=None
+        Class labels when `y_pred.shape = (n_samples, n_classes)`.
+        If not provided, labels will be inferred from `y_true`.
+
+    scale_by_half : bool or "auto", default="auto"
+        When True, scale the Brier score by half to lie in the [0, 1] range instead
+        of the [0, 2] range. The default "auto" option implements the rescaling to
+        [0, 1] only for binary classification (as customary) but keeps the
+        original [0, 2] range for multiclasss classification.
```

Comment:
Please add the `.. versionadded:: 1.7` directive to both new parameters' docstrings.

Reply:
done in c91aef1